man/ctags-optlib.7.rst.in

```diff
@@ -169,13 +169,19 @@ OPTION ITEMS
 	Define a kind for *<LANG>*.
 	Be not confused this with ``--kinds-<LANG>``.
 
-	*letter* must be an alphabetical character ('[0-9a-zA-EG-Z]')
+	*letter* must be an alphabetical character ('[a-zA-EG-Z]')
 	other than "F". "F" has been reserved for representing a file
 	since Exuberant-ctags.
 
-	*name* also must come from alphabetical characters ('[0-9a-zA-Z]+').
-	Do not use "file" as *name*. It has been reserved for representing
-	a file since Exuberant-ctags.
+	*name* must start with an alphabetical character or a number
+	character. The rest must be alphabetical characters
```

Comment:
I think this is reversed ... should be something like:

```
*name* must start with an alphabetic character, and the rest
must  be alphanumeric ('[a-zA-Z][a-zA-Z0-9]*').
```

Reply:
You are correct. Thank you. (Reflected the suggestion to the local source tree).